### PR TITLE
fix: Error: EACCES: permission denied, readlink '/usr/home'

### PR DESCRIPTION
**Pull request by avkarenow**
### 🔗 Linked issue

https://github.com/nuxt/nuxt/issues/15700

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Fix for a problem described in: https://github.com/nuxt/nuxt/issues/15700

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/rollup/config.ts [hunk 1]
```diff
@@ -329,7 +329,7 @@ export const plugins = [
               .filter((i) => typeof i === "string"),
           ],
           traceOptions: {
-            base: "/",
+            base: process.cwd(),
```
Comment (danielroe) [inline src/rollup/config.ts:332]: What about instead:

```suggestion
            base: nitro.options.rootDir,
```

Comment (avkarenow) [reply src/rollup/config.ts:332]:
```
base: nitro.options.rootDir,
```
works also and looks good.

**Comment by avkarenow** [comment]
> Thanks for PR. only is there a reproduction? ([nuxt/nuxt#15700](https://github.com/nuxt/nuxt/issues/15700) seems have not)

You can reproduce it on serv00.com hosting - just create a free account.

**Comment by avkarenow** [comment]
Is anything else needed to do the merge?

**Comment by pi0** [comment]
Hi. Sorry for checking late on this. We cannot merge the PR as is because of explained reasons above. I have moved a summary and workarounds to https://github.com/unjs/nitro/issues/1543. PR welcome for both either nft fixes or using common directory.

**Pull request closed without merging**